components/icu/src/lib.rs

```diff
@@ -173,6 +173,56 @@ pub mod decimal {
     pub use icu_decimal::*;
 }
 
+pub mod locale_canonicalizer {
+    //! This API provides functionality to canonicalize locale identifiers based
+    //! upon [`CLDR`] data.
+    //!
+    //! It currently supports the minimize and maximize likely subtags algorithms
+    //! as described in [`TR-35`].
```

Comment:
Nit: "TR 35" ("technical report") is legacy terminology; "UTS 35" ("unicode technical standard") is recommended instead.